### PR TITLE
add application overlays for knative

### LINKED ISSUE #476: add application overlays for knative
```
└── knative
    ├── knative-serving-crds
    └── knative-serving-install
```

### PR DESCRIPTION
**Pull request by kkasravi**
**Which issue is resolved by this Pull Request:**
Resolves #476

**Description of your changes:**
```
└── knative
    ├── knative-serving-crds
    └── knative-serving-install
```

**Checklist:**
- [ ] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate`
    3. `make test`

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/477)

### CODE REVIEW

File: knative/knative-serving-crds/overlays/application/application.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: knative-serving-crds
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: knative-serving-crds
+      app.kubernetes.io/instance: knative-serving-crds-v0.8.0
```
Comment (richardsliu) [inline knative/knative-serving-crds/overlays/application/application.yaml]: Why v0.8.0?

Comment (jlewi) [reply knative/knative-serving-crds/overlays/application/application.yaml]: Looks like that's the version of knative we are using.
https://github.com/knative/serving/releases

**Comment by jlewi** [comment]
/lgtm
/approve
/hold
@richardsliu Are you satisifed?

**Pull request merged**